doc/node.1

```diff
@@ -179,6 +184,47 @@ Write process warnings to the given
 .Ar file
 instead of printing to stderr.
 .
+.It Fl -diagnostic-report-directory
```

Comment:
Maybe all these sections should be placed between `-completion-bash` and `-enable-fips` sections alphabetically?